docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -38,112 +28,136 @@ If you install from the APT repository, then Grafana is automatically updated ev
 
-After you add the repository:
+   ```bash
+   sudo apt-get install -y apt-transport-https
+   sudo apt-get install -y software-properties-common wget
+   sudo wget -q -O /usr/share/keyrings/grafana.key https://apt.grafana.com/gpg.key
+   ```
 
-```bash
-sudo apt-get update
+1. To add a repository for stable releases, run the following commands:
 
-# Install the latest OSS release:
-sudo apt-get install grafana
+   ```bash
+   echo "deb [signed-by=/usr/share/keyrings/grafana.key] https://apt.grafana.com stable main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+   ```
 
-# Install the latest Enterprise release:
-sudo apt-get install grafana-enterprise
-```
+1. To add a respository for beta releases, run the following commands:
 
-### Install .deb package
+   ```bash
+   echo "deb [signed-by=/usr/share/keyrings/grafana.key] https://apt.grafana.com beta main" | sudo tee -a /etc/apt/sources.list.d/grafana.list
+   ```
 
-If you install the `.deb` package, then you will need to manually update Grafana for each new version.
+1. After you add the repository, run one of the following commands:
 
-1. On the [Grafana download page](https://grafana.com/grafana/download), select the Grafana version you want to install.
-   - The most recent Grafana version is selected by default.
-   - The **Version** field displays only finished releases. If you want to install a beta version, click **Nightly Builds** and then select a version.
-1. Select an **Edition**.
-   - **Enterprise** - Recommended download. Functionally identical to the open source version, but includes features you can unlock with a license if you so choose.
-   - **Open Source** - Functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
-1. Depending on which system you are running, click **Linux** or **ARM**.
-1. Copy and paste the code from the installation page into your command line and run. It follows the pattern shown below.
+   ```bash
+   sudo apt-get update
 
-```bash
-sudo apt-get install -y adduser
-wget <.deb package url>
-sudo dpkg -i grafana<edition>_<version>_amd64.deb
-```
+   # Install the latest OSS release:
+   sudo apt-get install grafana
 
-## Install from binary .tar.gz file
+   # Install the latest Enterprise release:
+   sudo apt-get install grafana-enterprise
+   ```
 
-Download the latest [`.tar.gz` file](https://grafana.com/grafana/download?platform=linux) and extract it. The files extract into a folder named after the Grafana version downloaded. This folder contains all files required to run Grafana. There are no init scripts or install scripts in this package.
+## Install Grafana using DEB or the standalone binaries
 
-```bash
-wget <tar.gz package url>
-sudo tar -zxvf <tar.gz package>
-```
+If you choose not to install Grafana using APT, you can download and install Grafana using DEB or the standalone binaries.
+
+### Before you begin
```

Comment:
@DanCech let's add prereqs if we have 'em.

Reply:
prereqs are already included in the download instructions

Reply:
will remove.